app/controllers/v0/preneeds/burial_forms_controller.rb

```diff
@@ -2,34 +2,37 @@
 module V0
   module Preneeds
     class BurialFormsController < PreneedsController
+      FORM = '40-10007'
+
       def new
       end
 
       def create
-        burial_form = ::Preneeds::BurialForm.new(burial_form_params)
-        validate!(burial_form)
+        forms = ::Preneeds::BurialForm.create_forms_array(burial_form_params)
+        validate!(forms)
 
-        resource = client.receive_pre_need_application(burial_form.message)
-        render json: resource, serializer: ReceiveApplicationSerializer
+        resources = forms.map { |form| client.receive_pre_need_application(form.as_eoas) }
+        render json: resources, each_serializer: ReceiveApplicationSerializer
       end
 
       private
 
       def burial_form_params
-        params.require(:pre_need_request)
-              .permit(
-                :application_status, :has_attachments, :has_currently_buried, :sending_code,
-                applicant: ::Preneeds::Applicant.permitted_params,
-                claimant: ::Preneeds::Claimant.permitted_params,
-                currently_buried_persons: [::Preneeds::CurrentlyBuriedPerson.permitted_params],
-                veteran: ::Preneeds::Veteran.permitted_params
-              )
+        params.require(:applications).map do |p|
+          p.permit(
+            :application_status, :has_attachments, :has_currently_buried, :sending_code,
+            applicant: ::Preneeds::Applicant.permitted_params,
+            claimant: ::Preneeds::Claimant.permitted_params,
+            currently_buried_persons: [::Preneeds::CurrentlyBuriedPerson.permitted_params],
+            veteran: ::Preneeds::Veteran.permitted_params
+          )
+        end
       end
 
-      def validate!(form)
-        # TODO: replace this schema once VetsJsonSchema preneeds is merged.
-        schema = JSON.parse(File.read(Settings.preneeds.burial_form_schema))
-        validation_errors = form.validate(schema, :pre_need_request)
+      def validate!(forms)
+        # schema = JSON.parse(File.read(Settings.preneeds.burial_form_schema))
+        schema = VetsJsonSchema::SCHEMAS[FORM]
+        validation_errors = ::Preneeds::BurialForm.validate(schema, forms)
```

Comment:
This validates each form? or just the first to encounter errors?

Reply:
@saneshark Currently it will stop at the first error. I was going to implement the multiple application error scenario as a separate PR after talking with Eugene/Mel to see how they want the error information returned, currently the FE cannot deal with multiple errors and applications...

Reply:
FWIW: ::Preneeds::BurialForm.validate(schema, forms) will validate all the forms at once - however, once I get clarification from the FE on how they want to handle errors, I can make the changes in the next PR, reserved for errors - I can either do it application by application, or en masse

Reply:
so normally we return an array of errors, I think the only things thats different in this case, is that you would need to have errors for broken down on an application by application basis. It really should all depend on how the params are provided. You should communicate the errors in precisely the same sort of JSON object.

In other words, if the array of application params comes in as:

```js
"applications": [ { "attribute1": "value"...}, {"attribute1": "value"...}]
```

errors would look similar:

```js
"errors": [ {"error": "some error message"}, {"error": "some error message"}]
```